index.d.ts

```diff
@@ -155,9 +270,9 @@ export interface FrozenProcessor {
    *   Nothing.
    */
   run(
-    node: Node,
+    node: ParseTree,
     file: VFileCompatible | undefined,
-    callback: RunCallback
+    callback: RunCallback<CompileTree>
```

Comment:
Would this be a `CompileTree`?
`run` doesn't have a compiler?

Or in this context does it mean, final tree type?

Reply:
`parse`s output is `ParseTree`
`run`s input is `ParseTree` and output is `CompileTree`
`stringify`s input is `CompileTree` and output is anything.

Open to better wording